ui/src/parseError.ts

```diff
@@ -8,64 +8,76 @@ import { IParsedError } from '../index';
 import { localize } from './localize';
 import { parseJson } from './utils/parseJson';
 
-// tslint:disable:no-unsafe-any
-// tslint:disable:no-any
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
```

Comment:
I feel like you should just disable these rules for the entire file like it was originally.  While Eric previously mentioned he doesn't like turning off the rules for more than one line at a time, errors are kind of in a different ballgame because they always get caught as an "any" type and the whole point of this file was to ensure that they would have a readable message for us to use.

Also Eric wrote this file originally so if he has a problem with 🤷‍♀️

Reply:
I agree with what Nathan said. Please just disable for the whole file

> Eric previously mentioned he doesn't like turning off the rules for more than one line at a time

That mostly just applies to production code. Test code can be a lot more flexible when it comes to linting

Reply:
Oh I didn't realize we _are_ talking about production code. In that case I agree less with what Nathan said, but I'm fine with it for this PR. Your main goal is to switch to eslint and keeping the status quo (disable for the whole file) is probably best in the short term. If/when we turn on lint in this file, we should fix some things rather than just disable each line. It'll take some work, but it might prevent something like this bug: https://github.com/microsoft/vscode-azuretools/pull/860